vcpkg/ports/oracle-instantclient/vcpkg.json

```diff
@@ -0,0 +1,6 @@
+{
+  "name": "oracle-instantclient",
+  "description": "Oracle Instant Client",
+  "homepage": "https://www.oracle.com/database/technologies/instant-client.html",
+  "version-string": "23.3.0.23.09"
```

Comment:
Shall we not advertise a different version (19) if we are on Mac ? or at least a comment

Reply:
A comment is not possible within json unfortunately. Another port (`oracle-instantlient-legacy-osx-x64`) to make it explicit could be done.

Reply:
> Another port (oracle-instantlient-legacy-osx-x64) to make it explicit could be done.

The more explicit the better, I think. But that's non blocking anyway

Reply:
I'm happy to accept contributions here, one of the drivers for vcpkg was to democratize the build process ;-)

Reply:
Fair point! 

I don't have a Mac environment to reproduce and contribute on this issue (and I don't want to, I have already have my part of painful proprietary software to deal with Oracle and Windows :man_shrugging:  ).

I'll try to bribe @Djedouas to work on this :)